source/percona-release.rst

```diff
@@ -45,12 +45,12 @@ official package:
       $ sudo apt-get update
       $ sudo apt-get install -y wget gnupg2 lsb-release
 
-1. Fetch the repository package:
+1. Fetch the repository package. You need to run this script as root or with :program:`sudo`. :
```

Comment:
why? it works well without root permissions

Reply:
Fixed.

Reply:
On the Ubuntu 20.04 (Docker) version I used to test, it was also missing CURL. Should I add that?